drivers/led/issi/is31fl3733-simple.c

```diff
@@ -126,6 +126,48 @@ bool is31fl3733_write_pwm_buffer(uint8_t addr, uint8_t *pwm_buffer) {
     return true;
 }
 
+void is31fl3733_init_drivers(void) {
+    i2c_init();
+
+#if !defined(IS31FL3733_SYNC_1)
+#    define IS31FL3733_SYNC_1 0
+#endif
+    is31fl3733_init(IS31FL3733_I2C_ADDRESS_1, IS31FL3733_SYNC_1);
+#if defined(IS31FL3733_I2C_ADDRESS_2)
+#    if !defined(IS31FL3733_SYNC_2)
+#        define IS31FL3733_SYNC_2 0
+#    endif
+    is31fl3733_init(IS31FL3733_I2C_ADDRESS_2, IS31FL3733_SYNC_2);
+#    if defined(IS31FL3733_I2C_ADDRESS_3)
+#        if !defined(IS31FL3733_SYNC_3)
+#            define IS31FL3733_SYNC_3 0
+#        endif
+    is31fl3733_init(IS31FL3733_I2C_ADDRESS_3, IS31FL3733_SYNC_3);
+#        if defined(IS31FL3733_I2C_ADDRESS_4)
+#            if !defined(IS31FL3733_SYNC_4)
+#                define IS31FL3733_SYNC_4 0
+#            endif
+    is31fl3733_init(IS31FL3733_I2C_ADDRESS_4, IS31FL3733_SYNC_4);
+#        endif
+#    endif
+#endif
```

Comment:
Could we not do something like this at the top of the file,
```
#if !defined(IS31FL3733_SYNC_1)
#    define IS31FL3733_SYNC_1 0
#endif
#if !defined(IS31FL3733_SYNC_2)
#    define IS31FL3733_SYNC_2 0
#endif
#if !defined(IS31FL3733_SYNC_3)
#    define IS31FL3733_SYNC_3 0
#endif
#if !defined(IS31FL3733_SYNC_4)
#    define IS31FL3733_SYNC_4 0
#endif
```

And these blocks could then just contain,
```suggestion

    is31fl3733_init(IS31FL3733_I2C_ADDRESS_1, IS31FL3733_SYNC_1);
#if defined(IS31FL3733_I2C_ADDRESS_2)
    is31fl3733_init(IS31FL3733_I2C_ADDRESS_2, IS31FL3733_SYNC_2);
#    if defined(IS31FL3733_I2C_ADDRESS_3)
    is31fl3733_init(IS31FL3733_I2C_ADDRESS_3, IS31FL3733_SYNC_3);
#        if defined(IS31FL3733_I2C_ADDRESS_4)
    is31fl3733_init(IS31FL3733_I2C_ADDRESS_4, IS31FL3733_SYNC_4);
#        endif
#    endif
#endif
```